quarkc/command.py

```diff
@@ -16,9 +16,9 @@
 Quark compiler.
 
 Usage:
-  quark [options] install [ --java | --python | --javascript | --all ] <file>...
-  quark [options] compile [ -o DIR ] [ --java | --python | --javascript | --all ] <file>...
-  quark [options] run ( --java | --python | --javascript ) <file>...
+  quark [options] install [ (--java | --python | --javascript | --ruby)... | --all ] <file>...
+  quark [options] compile [ -o DIR ] [ (--java | --python | --javascript | --ruby)... | --all ] <file>...
```

Comment:
To me, grouped variant is more accurate, i.e. `--java --all` is not a sensible usage. That said, I prefer the old style because it's shorter, easier to read, and easier to understand.